src/main/java/org/datadog/jenkins/plugins/datadog/model/BuildData.java

```diff
@@ -604,10 +604,8 @@ private boolean isGit(EnvVars envVars) {
      */
     public Map<String, Set<String>> getTags() {
         Map<String, Set<String>> allTags = new HashMap<>();
-        try {
+        if (DatadogUtilities.getTagsFromGlobalTags() != null){
```

Comment:
Assigning a `null` to a variable cannot trigger a `NullPointerException`.
The corollary to this is that if `allTags = DatadogUtilities.getTagsFromGlobalTags()` throws an NPE, then `DatadogUtilities.getTagsFromGlobalTags() != null` throws an NPE as well.

This change looks like it might actually break things (provided the `catch` block was actually needed, and not placed there "just in case").

Reply:
Ok sounds good, I have reverted the code.